pkg/client/unversioned/request.go

```diff
@@ -759,8 +759,10 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 	// Did the server give us a status response?
 	isStatusResponse := false
 	var status unversioned.Status
-	if err := r.codec.DecodeInto(body, &status); err == nil && status.Status != "" {
-		isStatusResponse = true
+	if r.codec != nil {
```

Comment:
[This] does not have a codec https://github.com/kubernetes/kubernetes/pull/14592/files#diff-f013b1aafa99ed0c479d939400d636e9R408.
This UnversionedRESTClient is used to discovery APIs by visiting /apis, the message returned by the API server is not coded.